### PR TITLE
Auditing executeWrite

### PR DESCRIPTION
**Pull request by ankitmashu**
-- Execute Write Code
-- Test Cases
-- DataBrokerImpl publishmessage
-- Databroker Test Cases

### CODE REVIEW

**Review by kailash — APPROVED** [review]
LGTM

File: src/main/java/iudx/file/server/auditing/util/Constants.java [hunk 1]
```diff
@@ -27,4 +27,12 @@ public class Constants {
   public static final String  DATABASE_TABLE_NAME = "databaseTableName";
   public static final String WRITE_QUERY =
           "INSERT INTO $0 (id,api,userid,epochtime,resourceid,isotime,providerid,size) VALUES ('$1','$2','$3',$4,'$5','$6','$7',$8)";
+  public static final String ORIGIN_SERVER = "file-server";
+  public static final String ORIGIN = "origin";
+  public static final String PRIMARY_KEY= "primaryKey";
+  public static final String EPOCH_TIME = "epochTime";
+  public static final String ISO_TIME = "isoTime";
+  public static final String EXCHANGE_NAME = "auditing";
+  public static final String ROUTING_KEY = "##";
```
Comment (gopal-mahajan) [inline src/main/java/iudx/file/server/auditing/util/Constants.java:36]: routingKey is # not ##

Comment (ankitmashu) [reply src/main/java/iudx/file/server/auditing/util/Constants.java]: > routingKey is # not ##
Solved

File: src/test/java/iudx/file/server/databroker/DataBrokerServiceImplTest.java [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+                return null;
+              }
+            })
+        .when(rmqClientMock)
+        .basicPublish(anyString(), anyString(), (Buffer) any(), any());
+
+    databroker.publishMessage(
+        new JsonObject(),
+        "auditing",
+        "##",
```
Comment (kailash) [inline src/test/java/iudx/file/server/databroker/DataBrokerServiceImplTest.java:113]: routing key?

Comment (ankitmashu) [reply src/test/java/iudx/file/server/databroker/DataBrokerServiceImplTest.java:113]: > routing key?

Solved

**Review by kailash — APPROVED** [review]

File: src/main/java/iudx/file/server/auditing/AuditingServiceImpl.java [hunk 3]
```diff
@@ -1,42 +1,43 @@
...
-  private final QueryBuilder queryBuilder = new QueryBuilder();
-  private JsonObject query = new JsonObject();
+  private JsonObject writeMessage = new JsonObject();
   private String databaseIP;
   private int databasePort;
   private String databaseName;
   private String databaseUserName;
   private String databasePassword;
   private String databaseTableName;
   private int databasePoolSize;
```
Comment (gopal-mahajan) [inline src/main/java/iudx/file/server/auditing/AuditingServiceImpl.java:40]: what is the use of this declaration. not being used

Comment (ankitmashu) [reply src/main/java/iudx/file/server/auditing/AuditingServiceImpl.java]: > what is the use of this declaration. not being used

Solved.

File: src/main/java/iudx/file/server/auditing/AuditingServiceImpl.java [hunk 4]
```diff
@@ -1,120 +1,61 @@
...
-  private String databaseName;
-  private String databaseUserName;
-  private String databasePassword;
-  private String databaseTableName;
-  private int databasePoolSize;
-  private ResponseBuilder responseBuilder;
+  private final ObjectMapper objectMapper = new ObjectMapper();
+  private JsonObject writeMessage = new JsonObject();
 
   public AuditingServiceImpl(JsonObject propObj, Vertx vertxInstance) {
```
Comment (gopal-mahajan) [inline src/main/java/iudx/file/server/auditing/AuditingServiceImpl.java:28]: what is the use of propObj ?

**Review by gopal-mahajan — APPROVED** [review]
LGTM

**Pull request merged**